assets/src/edit-story/components/panels/sizePosition.js

```diff
@@ -142,10 +157,14 @@ function SizePositionPanel({
       if (lockAspectRatio) {
         const ratio = oldWidth / oldHeight;
         if (!isResizeWidth) {
-          return { width: dataPixels(newHeight * ratio) };
+          return {
+            width: setMinMax(dataPixels(newHeight * ratio), MIN_MAX.WIDTH),
+          };
         }
         if (!isResizeHeight) {
-          return { height: dataPixels(newWidth / ratio) };
+          return {
+            height: setMinMax(dataPixels(newWidth * ratio), MIN_MAX.HEIGHT),
+          };
         }
```

Comment:
Hm, I guess these should still follow proportions, right? If I have a 100x200 element with proportions locked and resize it to 1000 width - wouldn't the expected result be 500x1000, as that is the maximum allowed size still following proportions?

Reply:
What happens now is, that the elements resizes to 1000x1000, which is not following proportions.

Reply:
Yeah @barklund it should follow the proportions, thanks for the spec heads up. Let me check this behavior right now.

Reply:
@merapi replying here versus slack chat. If the aspect ratio is locked, it should be 500x1000.